tcms/tests/__init__.py

```diff
@@ -227,3 +227,23 @@ def setUpTestData(cls):
         cls.execution_4 = executions[3]
         cls.execution_5 = executions[4]
         cls.execution_6 = executions[5]
+
+
+class CreateTestUsers(LoggedInTestCase):
```

Comment:
I think that maybe you got the wrong idea on how this class will be used (or maybe I got the wrong idea, but here's my two views on how this should look and most importantly how it should be used):

this will be a base test class, which will provide helper methods. the other test classes (the ones that actually contains tests) will inherit from it, and will use these methods, something like
```python
class CreateTestUser(LoggedInTestCase):
    def user_has_only(self, tested_permission):
        for perm in Permission.objects.all():
            remove_perm_from_user(self.tester, perm)

        user_should_have_perm(self.tester, tested_permission)

   def user_is_missing_only(self, tested_permission):
       for perm in Permission.objects.all():
           remove_perm_from_user(self.tester, perm)

       user_should_have_perm(self.tester, tested_permission)

class TestTelemetry(CreateTestUser):
    def test_user_with_permission_sees_the_page(self):
        self.user_has_only('perm_to_see_this_page')
        ....
        self.assertEquals(response.status_code, 200)

    def test_user_without_perm_does_not_see_the_page(self):
        self.user_is_missing_only('perm_to_see_this_page')
        ....
        self.assertEquals(response.status_code, 403)
```
with these thinks in mind, you should change a few things in this PR:

1. remove the `@classmethod` decorations from the two helpers methods, because they won't be used like class methods, but as instance methods
2. wherever you want to use them, instead of calling `CreateTestUsers.create....` inherit from `CreateTestUsers` and call `self.create...`
3. also, naming - the current method names are too verbose, and misleading (you don't create users, you just remove and add permissions), change them to the one suggested by Alex (and change from camelCase to snake_case as this is the Python convention for method and variable names)

example for that:
you want to use the new methods in `TestCreateTestUsers`. this class inherits from `BaseCaseRun`, which inherits from `BasePlanCase`, which inherits from `LoggedInTestCase`
so in order to use the new methods, you should make `BasePlanCase` inherit from `CreateTestUsers`.
so 
`TestCreateTestUsers -> BaseCaseRun -> BasePlanCase -> LoggedInTestCase` becomes 
`TestCreateTestUsers -> BaseCaseRun -> BasePlanCase -> CreateTestUsers -> LoggedInTestCase`
the change of breaking something with change is small, because `CreateTestUsers` inherits from `LoggedInTestCase` and does not change the behaviour of it

also, I wonder if we can think of a better name for `CreateTestUsers`